docs/snippets/html/button-story-default-exports.js.mdx

```diff
@@ -0,0 +1,13 @@
+```js
+// Button.stories.js
+
+import { createButton } from './Button';
+
+export default {
+  /* 👇 The title prop is optional.
+  * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+  * to learn how to generate automatic titles
+  */
+  title: 'Button',
```

Comment:
@Matthew-Smith if you could update the comment text to:
```js
/* 👇 The title prop is optional.
  * See https://storybook.js.org/docs/html/configure/overview#configure-story-loading
  * to learn how to generate automatic titles
  */
```
And apply it to the other comments in this pr I'd appreciate it.

The reasoning behind this is so that we give them the proper location as they're using the `html` version of it. Not the React version